### PR TITLE
add job sites, search, aplhabetical order

### PR DESCRIPTION
**Pull request by rkimera94**

### CODE REVIEW

**Comment by rkimera94** [comment]
I have made some changes to the job list. I added some values, added a search and also ordered them alphabetically. 
@breck7 @celtic-coder

File: site/lists/jobs.scroll [hunk 1]
```diff
@@ -6,30 +6,38 @@ columnWidth 60
...
+ Dribbble|https://dribbble.com/jobs?location=Anywhere
+ Flexjobs|https://www.flexjobs.com/ 
  GitHub Jobs|https://jobs.github.com/
  Glassdoor|https://www.glassdoor.com/
  Hacker News Jobs|https://news.ycombinator.com/jobs
+ Hubstaff|https://talent.hubstaff.com/
  Indeed|http://indeed.com/
+ Jobspresso|https://jobspresso.co/
  LinkedIn|https://www.linkedin.com/
+ Monster|https://www.monster.com/
```
Comment (breck7) [inline site/lists/jobs.scroll:23]: of course! we should probably look and see if they have API integrations we could use

File: site/lists/jobs.scroll [hunk 2]
```diff
@@ -6,30 +6,38 @@ columnWidth 60
...
+ Otta|https://otta.com/
```
Comment (breck7) [inline site/lists/jobs.scroll:24]: interesting!

**Comment by breck7** [comment]
This is great work @rkimera94 ! Live now. How can we make this page better? What if we added a "year" column?

File: site/lists/jobs.scroll (hunk 1 above)
Comment (celtic-coder) [reply site/lists/jobs.scroll:23]: Yes, Monster is the golden-oldie of online job sites!

**Comment by rkimera94** [comment]
welcome @breck7  and @celtic-coder .Thanks for the feedback

**Comment by rkimera94** [comment]
About the year column? is it supposed to be the year the platform was created?

File: site/lists/jobs.scroll (hunk 1 above)
Comment (rkimera94) [reply site/lists/jobs.scroll:23]: okay, let me check if they have APIs

**Pull request merged**